Assets/Mirror/Weaver/Weaver.cs

```diff
@@ -179,7 +179,6 @@ class Weaver
         public static MethodReference sendEventInternal;
         #endregion
 
-        public static AssemblyDefinition m_UnityAssemblyDefinition;
```

Comment:
Where are you declaring UnityAssembly?

Reply:
307612c

Reply:
sorry, I see now I shouldn't have put those new declarations in 307612c, they should be in each of these individual commits so can literally see them moving in there. Each commit leaves the library in a valid state, but it doesn't exactly make sense that you cant see UnityAssembly is just a few lines above here, since it wasn't put there at the same time. I'll do better than that in future PRs

Reply:
you can just rebase and fix all those commits.

Reply:
mercurial doesn't have such magic potions as rebase or cherry pick etc. I'll figure that out though

Reply:
indeed,  that is why git has such a commanding lead in marketshare.

Reply:
i find mercurial much simpler to use and its internals seem less complex. also it was very seamless a transition for me from SVN, and bitbucket lets me do all the private work I want (though granted I could put git projects there). private repos on github is fairly new. But I'm no holy warrior or anything. i hold nothing sacred ... tabs, spaces, braces, whatev ... except keeping CRLF the fuck out of my repo and LF the fuck out of my visual studio xD. obviously both versioning systems are perfectly capable. I've been meaning to get better at git, so now I get my wish lol

Reply:
I used both for many years.   No way I will ever go back to mercurial.  Yes,  it is simpler, but it is a toy next to git.